Source/SwiftLintFramework/Extensions/File+SwiftLint.swift

```diff
@@ -161,17 +161,20 @@ extension File {
         }.map { $0.0 }
     }
 
+    internal typealias MatchMapping = (NSTextCheckingResult) -> NSRange
```

Comment:
Why did you add this, the `exclusionMapping` parameter and replace the `ranges()` call later? Seems like the previous code does the exact same thing?